xwiki-platform-core/xwiki-platform-comment/xwiki-platform-comment-api/src/main/java/org/xwiki/platform/comment/Comment.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.platform.comment;
+
+import java.util.Date;
+
+import org.xwiki.user.UserReference;
+
+public interface Comment
```

Comment:
Is it generic enough to also handle annotation? For instance displaying the quote.
Also missing since and comment but I guess this is in a draft state.

Reply:
> Is it generic enough to also handle annotation? For instance displaying the quote.

the fact that we use comment for annotation is an implementation detail for me: we should have a dedicated interface for the annotation, there's no need to reuse that for annotation.

Reply:
But we display annotations in the comments UI, and they have a special display.